07-routing-gossip.md

```diff
@@ -432,12 +432,6 @@ fields in the `channel_update` message:
 | ------------- | ------------------------- | -------------------------------- |
 | 0             | `option_channel_htlc_max` | `htlc_maximum_msat`              |
 
-Note that the `htlc_maximum_msat` field is static in the current
```

Comment:
Why not also remove lines 428 to 433 entirely as well? We'll never want to use those flags in the future, we'll use tlvs instead, so just say that this whole `message_flags` field must be hard-coded to 1 (we could even go further and rename it to show that it's just an obsolete constant)?

Reply:
Done.